docs/books/admin_guide/08-process.md

```diff
@@ -314,17 +314,31 @@ The command `nice` allows the execution of a command by specifying its priority.
 nice priority command
 ```
 
-Example:
+Usage example:
 
 ```bash
-nice -n+15 find / -name "file"
+nice --adjustment=-5 find / -name "file"
+
+nice -n -5 find / -name "file"
+
+nice --5 find / -name "file"
```

Comment:
-5 != --5 

@jimcat8

Reply:
```
nice nice --3 nice
7
```

Reply:
@wsoyinka Are you still waiting for a change on this?